_posts/2021-05-10-helm-charts-for-wildfly.adoc

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: Helm Charts for WildFly
+date: 2021-05-10
+tags: wildfly openshift helm
+synopsis: Helm Charts for WildFly simplifies deployment of WildFly applications on OpenShift.
+author: jmesnil
+---
+
+https://helm.sh[Helm] is a tool that help define, install and upgrades complex applications on Kubernetes clusters.
+
+We have now released a Helm Chart for WildFly to help our users build and deploy WildFly applications on OpenShift that leverages all the innovative features provided by the WildFly ecosystem.
+
+The `wildfly` Chart can be used to achieve two objectives:
+
+* build an application image from a Git repository containing a Java project. This step leverages WildFly features such as Galleon provisioning or Bootable Jar to provide the fittest image for your application requirements.
+* deploy an application image with all the resources and configuration needed to integrate it on OpenShift.
+
+By default, the two steps are connected: the `build` step's output is an application image that becomes the input of the `deploy` step.
+However it is possible to enable or disable these steps: if you have another way to build your application image, you can disable the `build` step and still deploys it by specifying where your application image can be pulled.
```

Comment:
s/However it/However, it

Reply:
Not sure about this one, but it sounds strange with the 's':

s/step and still deploys it/step and still deploy it